content/_code-samples/use-tickets/js/use-tickets-multisig.js

```diff
@@ -0,0 +1,129 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
```

Comment:
```suggestion
if (typeof module !== "undefined") {
    // Use var here because const/let are block-scoped to the if statement.
    var xrpl = require('xrpl')
}
```

Indentation doesn't look correct.